packages/tasks/src/model-libraries.ts

```diff
@@ -116,6 +116,15 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoName: "doctr",
 		repoUrl: "https://github.com/mindee/doctr",
 	},
+	elm: {
+		prettyLabel: "ELM",
+		repoName: "elm",
+		repoUrl: "https://github.com/slicex-ai/elm-v0.1",
```

Comment:
What does `0.1` refers to in `elm-v0.1`? Asking because if the repo/library will change between versions, then I think it'd be good to tag models as `elm-v0.1` instead of `elm`. What will happen once an `elm-v0.2` or `elm-v1.0` family of models will be shipped? If library will not change then I'd suggest to rename the github repo to `slicex-ai/elm`. But having a mixture of both versioned library and not versioned tag is not good IMO.

Reply:
Thanks @Wauplin! We have renamed the github repo to `slicex-ai/elm` and pushed the changes to the PR.